app/views/clients/va/_consent_form_flags.haml

```diff
@@ -0,0 +1,3 @@
+.client-consent-status.mb-4
+  %h3.mt-0 Consent
```

Comment:
I wonder if this should be `Release of Information` or something other than just `Concent` (maybe client consent).  We've historically referred to the release, but we've also been inconsistent.